test/fixtures/export/Do not caps-protect name fields #384.noopsort.bibtex

```diff
@@ -179,17 +179,17 @@
 }
 
 @book{este,
-  author = {\noopsort{Este}d'\relax{} Este, Beatrice},
+  author = {{\noopsort{este}}d'\relax Este, Beatrice},
   note = {[Este][Beatrice d']}
 }
 
 @book{medici,
-  author = {\noopsort{Medici}de'\mbox{} Medici, Lorenzo},
-  note = {[Medici][Lorenzo de'\mbox]}
+  author = {{\noopsort{medici}}de' Medici, Lorenzo},
+  note = {[Medici][Lorenzo de'U+200B]}
 }
 
 @book{al-hakim,
-  author = {\noopsort{Hakim}{al-\relax{} Hakim}, Tawfiq},
+  author = {{\noopsort{hakim}}{al-\relax Hakim}, Tawfiq},
```

Comment:
Sorry, those are my test results: `\relax{}<SPACE>` does not work, it has to be `\relax<SPACE>`